src/middleware/github-server-app-middleware.ts

```diff
@@ -36,29 +41,23 @@ export const GithubServerAppMiddleware = async (req: Request, res: Response, nex
 
 		req.log.info("Found server app for installation. Defining GitHub app config for GitHub Enterprise Server.");
 
-		//TODO: ARC-1515 decide how to put `gitHubAppId ` inside `gitHubAppConfig`
-		res.locals.gitHubAppId = gitHubServerApp.id;
 		res.locals.gitHubAppConfig = {
 			gitHubAppId: gitHubServerApp.id,
 			appId: gitHubServerApp.appId,
 			uuid: gitHubServerApp.uuid,
-			hostname: gitHubServerApp.gitHubBaseUrl,
-			clientId: gitHubServerApp.gitHubClientId,
-			gitHubClientSecret: await gitHubServerApp.getDecryptedGitHubClientSecret(),
```

Comment:
I look through the code and find ONLY ONE place that use the `locals.gitHubAppConfig.privateKey`.
So I remove the secrets here, not even put a function closure to decrypt it.

Can refactor this later when we really need to grab the secrets from `res.locals`.